dags/main_summary.py

```diff
@@ -94,10 +94,22 @@
                       uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/mozetl/testpilot/txp_mau_dau.py",
                       dag=dag)
 
+t9 = EMRSparkOperator(task_id="p2h_main_summary",
+                      job_name="P2H for Main Summary",
+                      execution_timeout=timedelta(hours=4),
```

Comment:
Will this timeout be sufficient? @fbertsch has just mentioned 5+ hours as a possibility.